pkg/webhook/webhook.go

```diff
@@ -178,6 +180,28 @@ func (w *Config) createFleetWebhookConfiguration(ctx context.Context) error {
 					},
 				},
 			},
+			{
+				Name:                    "fleet.customresourcedefinition.validating",
+				ClientConfig:            w.createClientConfig(v1.CustomResourceDefinition{}),
+				FailurePolicy:           &failPolicy,
+				SideEffects:             &sideEffortsNone,
+				AdmissionReviewVersions: []string{"v1", "v1beta1"},
+				Rules: []admv1.RuleWithOperations{
+					{
+						Operations: []admv1.OperationType{
+							admv1.Create,
+							admv1.Update,
+							admv1.Delete,
+						},
+						Rule: admv1.Rule{
+							APIGroups:   []string{"apiextensions.k8s.io"},
```

Comment:
Hi Arvind! Another nit here: since we are using references instead of hard-coded values in the actual handler logic; do you want to apply it here as well? Or course I am also OK with top-level constants if you may.